docs/user/subprojects.rst

```diff
@@ -1,11 +1,17 @@
-Subprojects
-===========
+Organizing projects in one site: Subprojects
```

Comment:
I think it's okay to capitalize after a `:`? :)

I was also thinking about saying "multiple" or "many" here but I wanted to trim it because of length. But now that you mention it, I feel confident to add it :+1:

Reply:
(edited your suggestion to capital Subprojects)